src/EnergyPlus/MoistureBalanceEMPDManager.cc

```diff
@@ -249,9 +249,9 @@ void GetMoistureBalanceEMPDInput(EnergyPlusData &state)
 
         auto const &constr = state.dataConstruction->Construct(surf.Construction);
         auto const *mat = dynamic_cast<const MaterialEMPD *>(s_mat->materials(constr.LayerPoint(constr.TotLayers)));
-        assert(mat != nullptr);
+        // assert(mat != nullptr);
```

Comment:
The way things are structured now, it's totally fine for this to result in a nullptr.  If the material doesn't have any EMPD properties, that's fine.  So we shouldn't do a hard assertion.